code/modules/mining/abandoned_crates.dm

```diff
@@ -7,14 +7,14 @@
 	icon_state = "securecrate"
 	var/code = null
 	var/lastattempt = null
+	var/lastattempt2 = null
```

Comment:
you literally replaced ``lastattempt```'s usage everywhere with this, so why is the ```replacetext(input, 0, "z")``` not performed on the original ```lastattempt``` var instead of adding a new one?